test/lib/flat_mutation_reader_assertions.hh

```diff
@@ -736,18 +773,40 @@ public:
         return *this;
     }
 
-    flat_reader_assertions_v2& produces_range_tombstone_change(const range_tombstone_change& rt) {
-        testlog.trace("Expect {}", rt);
-        auto mfo = read_next();
-        if (!mfo) {
-            BOOST_FAIL(format("Expected range tombstone {}, but got end of stream", rt));
+    flat_reader_assertions_v2& may_produce_tombstones(position_range range) {
+        testlog.trace("Expect possible range tombstone changes in {}", range);
+        while (auto next = peek_next()) {
+            if (!next->is_range_tombstone_change()) {
+                break;
+            }
+            position_in_partition::less_compare less(*_reader.schema());
+            auto rtc = maybe_drop_deletion_time(next->as_range_tombstone_change());
+            if (less(rtc.position(), range.start()) || less(range.end(), rtc.position())) {
```

Comment:
AFAIK there should be a `range.contains()` that you can use for this.

Reply:
There is no existing range predicate (that I could find) that is inclusive on both ends.  Perhaps I should have introduced one?

Reply:
looks like it is, yes, thanks for the pointer